charts/blackduck-connector/templates/secrets.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+data:
+  {{ .Values.blackduck.connectionsEnvironmentVariableName }}: {{ .Values.secret.externalHub | toJson | b64enc }}
+  securedRegistries.json: {{ .Values.secret.securedRegistries | toJson | b64enc }}
```

Comment:
you have tested with securedRegistries right??

Reply:
and also external hub scenario?

Reply:
Both tested :) Senthil, my artifactory and quay processors won't work without it :)